src/rust/ide/src/controller/searcher.rs

```diff
@@ -59,6 +59,11 @@ pub struct NotASuggestion {
     index : usize,
 }
 
+#[allow(missing_docs)]
+#[fail(display="An action is not supported in current environment.")]
```

Comment:
It would be really nice if we could include here information *what* action is not supported and *why* ("current environment" does not really say much).
Perhaps `manage_projects` should return a `Result`?